packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -1200,6 +1203,13 @@ export abstract class Chart extends Observable implements AgChartInstance {
         }
     }
 
+    protected onMouseDown(event: InteractionEvent<'mouse-down'>) {
+        this.mouseDownIds = undefined;
+        this.checkSeriesNodeRange(event, (series, datum) => {
```

Comment:
How expensive is this operation? If expensive, maybe we could just use some measurement of distance between the mouse down and mouse up events to determine if a 'click' should be generated?

Reply:
I thought about using `mousemove` or distance here, but I don't think it is universally applicable as some nodes are a few pixels across and some are half the chart.

In terms of performance, I don't think adding one more call to this will make a noticeable difference. But I believe it is `O(n)` to the series + nodes + pick modes?